tests/test_jobs.py

```diff
@@ -146,13 +145,15 @@ def test_create_spark_job(client, mocker, notebook_maker,
         size=spark_job.size,
         user_email=test_user.email,
     )
+    spark_job.latest_run = spark_job.get_latest_run()
```

Comment:
Why do you set the `latest_run` attribute here with the result of `get_latest_run`?